README.md

```diff
@@ -1,4 +1,4 @@
-`<h1 align="center">LeetDroid - An android client for LeetCode</h1>
```

Comment:
sure, even though the change is small but I would take it. You can revert the changes other than this :)